pkg/framework/plugins/nodeutilization/normalizer/normalizer.go

```diff
@@ -84,17 +84,15 @@ func Min[K comparable, N Number, V ~map[K]N](values []V, limit N) []V {
 	return result
 }
 
-// Deviate receives an object and a list of "deviations", returns an slice with
-// returns an slice of the same input where the internal value is added to each
-// deviation. Negative deviations are allowed.
-func Deviate[K comparable, N Number, V ~map[K]N](values V, deviations []V) []V {
-	result := []V{}
-	for _, dev := range deviations {
-		current := V{}
-		for name, value := range values {
-			current[name] = value + dev[name]
+// Sum receives an object and a "deviation", returns an slice with the internal
+// values "summed" to each deviation.
+func Sum[K comparable, N Number, V ~map[K]N](values, deviation V) V {
+	result := V{}
+	for name, value := range values {
+		if _, ok := deviation[name]; !ok {
+			continue
 		}
-		result = append(result, current)
+		result[name] = value + deviation[name]
```

Comment:
s/values, deviation/mapA, mapB/ or similar naming to make the function even more generic.

Reply:
Done.